opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -1345,10 +1345,15 @@ def test_to_json(self):
             is_remote=False,
             trace_flags=trace_api.TraceFlags(trace_api.TraceFlags.SAMPLED),
         )
-        parent = trace._Span("parent-name", context, resource=Resource({}))
+        parent = trace._Span(
+            "parent-name",
+            context,
+            resource=Resource({"hello": "world"}),
```

Comment:
nit: do we want to keep these resource attributes since they don't appear to have an effect on this test?